clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/OAuthBearerSaslClientProvider.java

```diff
@@ -27,7 +27,7 @@ public class OAuthBearerSaslClientProvider extends Provider {
 
     @SuppressWarnings("this-escape")
     protected OAuthBearerSaslClientProvider() {
-        super("SASL/OAUTHBEARER Client Provider", 1.0, "SASL/OAUTHBEARER Client Provider for Kafka");
+        super("SASL/OAUTHBEARER Client Provider", "1.0", "SASL/OAUTHBEARER Client Provider for Kafka");
         put("SaslClientFactory." + OAuthBearerLoginModule.OAUTHBEARER_MECHANISM,
```

Comment:
`new Provider(String, double, String)` has been deprecated since Java 9.
Document recommend to use `new Provider(String, String, String)` instead.